### DISCUSSION TITLE
Designer Antd基本流程浅析

### DISCUSSION THREAD

**Discussion by febugcoder** [alibaba/formily#1961] [Show and tell]
主要是为了理清楚整体流程，在基于Designer二次开发的时候容易理解一些。
有错误希望大佬们指出，感谢~
![designable源码](https://user-images.githubusercontent.com/41506360/128444644-b7e67956-ba8d-4d93-8a5d-f33df4b5e3c1.png)

**Comment by janryWang** [comment]
提到的地方基本上理解都没问题，不过还是不够全哈

**Reply by febugcoder** [reply]
确实，特别是Engine设计器引擎这块，这里主要是为了基于Designer二次开发时用起来顺利一些梳理的基本流程。
期待文档以及设计思路，非常感谢大佬回复~
